.atomist/handlers/command/issue/LabelIssue.ts

```diff
@@ -63,10 +58,10 @@ class RemoveLabelIssueCommand implements HandleCommand {
 
     public handle(ctx: HandlerContext): CommandPlan {
         const plan = new CommandPlan();
-        const exec = execute("remove-label-github-issue", this);
-        plan.add(handleErrors(exec, this));
+        const ex = execute("label-github-issue", this);
```

Comment:
It is not at all clear to me how, when calling label-github-issue, it knows whether to add or remove it.

Reply:
It is effectively a toggle. In the rug function it checks to see if the label is already there - if it is the label is removed, if not it is added

Reply:
Perhaps the intent should be "toggle issue label" then?

Reply:
ok. will make the changes